tmt/steps/execute/__init__.py

```diff
@@ -499,7 +504,10 @@ def go(self) -> None:
                     phase.go()
 
                 elif isinstance(phase, ExecutePlugin):
-                    phase.go(guest)
+                    # TODO: re-injecting the logger already given to the guest,
```

Comment:
Maybe each added `TODO` item could be accompied with a `tmt` issue link?

Reply:
In general, I agree, in this case, I wouldn't waste time on an issue, as this TODO should go away when we start completing all those patches together. It's here more or less for current readers, and it will not survive long enough to warrant an issue :)

Reply:
agreed, ty